### PR TITLE
[TASK] Disable github settings by default

### PR DESCRIPTION
**Pull request by liayn**

### CODE REVIEW

File: Documentation/Settings.cfg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # #####
 #
 # Settings.cfg - A TYPO3 Documentation Project's Configuration File
-# Information about Settings.cfg: 
+ # Information about Settings.cfg:
```
Comment (DanielSiepmann) [inline Documentation/Settings.cfg:4]: I guess this was an accident?

File: Documentation/Settings.cfg [hunk 2]
```diff
@@ -39,8 +39,8 @@ copyright   = by {extension.extensionKey}
 # ...  (recommended) to get the "Edit me on Github Button"
 # .................................................................................
 
-github_branch        = master
-github_repository    = TYPO3-Documentation/TYPO3CMS-Example-ExtensionManual
+#github_branch        = master
```
Comment (DanielSiepmann) [inline Documentation/Settings.cfg:42]: Should we add a hint to comment this out? As this is an example for every beginner to start with, we should explain the necessary steps and don't assume any prior knowledge. What do you think?

File: Documentation/Settings.cfg (hunk 1 above)
Comment (liayn) [reply Documentation/Settings.cfg:4]: Indeed

File: Documentation/Settings.cfg (hunk 2 above)
Comment (liayn) [reply Documentation/Settings.cfg:42]: What do you have in mind? A description what "edit me on github" does?

Comment (DanielSiepmann) [reply Documentation/Settings.cfg:42]: Just a hint that `#` is a comment, and that authors should remove the comment and adjust the repository.

Comment (liayn) [reply Documentation/Settings.cfg:42]: Hm... I honestly think this is something we can expect from extension authors to recognize. Simply by scrolling over the file you immediately spot that free text is always beginning with a `#`.

**Comment by liayn** [comment]
@DanielSiepmann ping

**Pull request merged**